### PR TITLE
Manager main page view

### PR DESCRIPTION
**Pull request by lexussuxel**
fronted part 
image
image
(Figma example)

### CODE REVIEW

File: packages/frontapp/src/app/components/ManagerInfoCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+}
+
+const WrapperCard = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background: ${colors.bg_secondary};
+  height: inherit;
+  width: 100%;
+  box-shadow: 0px 10px 70px rgba(26, 30, 214, 0.08);
```
Comment (unrealsolver) [inline packages/frontapp/src/app/components/ManagerInfoCard/index.tsx:25]: Please use constants for colors

File: packages/frontapp/src/app/components/ManagerInfoCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+`;
+
+function renderSwitch(type: ManagerCardTypes) {
+  switch (type) {
+    case ManagerCardTypes.OVERDUE:
+      return 'The following users have not turned in their books';
+    case ManagerCardTypes.DONATES:
+      return 'New arrivals awaiting your confirmation';
+    case ManagerCardTypes.NOTIFICATIONS:
+      return 'Problems faced by users';
```
Comment (unrealsolver) [inline packages/frontapp/src/app/components/ManagerInfoCard/index.tsx:147]: Please add localization template here,

File: packages/frontapp/src/utils/managerCardTypes.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export enum ManagerCardTypes {
+  OVERDUE = 'Overdue',
+  DONATES = 'Donates',
+  NOTIFICATIONS = 'New_user_notification',
+}
```
Comment (unrealsolver) [inline packages/frontapp/src/utils/managerCardTypes.ts:5]: This is not 'utilitary' data type, it's related to the domain/model, so it should be defined in appropriate place (usually near the component it used by).
Also, it's not necessary to define descriptions for enum values, just use keys.

File: packages/ui-kit/src/lib/colors/index.ts [hunk 4]
```diff
@@ -9,8 +9,10 @@ export const colors = {
   main_black: '#333333',
   warning_yellow: '#FF910F',
   problem_red: '#FF372A',
+  red_main: '#FF372A',
   main_green: '#2AA816',
   bg_primary: '#F3F7FF',
   bg_secondary: '#FFFFFF',
+  bg_fields: '#EFF4FF',
```
Comment (unrealsolver) [inline packages/ui-kit/src/lib/colors/index.ts:16]: Usually, we use camel case. I don't know why other constants are defined in snake case

File: packages/ui-kit/src/lib/colors/index.ts [hunk 5]
```diff
@@ -9,8 +9,10 @@ export const colors = {
   main_black: '#333333',
   warning_yellow: '#FF910F',
   problem_red: '#FF372A',
+  red_main: '#FF372A',
```
Comment (unrealsolver) [inline packages/ui-kit/src/lib/colors/index.ts:12]: It's exactly the same as the color above. Are they separated intentionally?

**Review by unrealsolver — CHANGES_REQUESTED** [review]

File: packages/ui-kit/src/lib/colors/index.ts (hunk 5 above)
Comment (lexussuxel) [reply packages/ui-kit/src/lib/colors/index.ts:12]: Didn't notice that

File: packages/frontapp/src/app/pages/HomePage.tsx [hunk 6]
```diff
@@ -17,32 +22,127 @@ const WrapperHome = styled.div`
...
+      title: 'Alica in Wonderland',
+      img: Pic,
+      person_id: '1',
+      created_at: '2022-08-01 10:45:09.999641',
+      description:
+        '2lala lalal alallala lalal lla lal lala la   lla lal lala la   lla lal lala la   lla lal lala la   lla lal lala la  lla lal lala la  lla lal lala la  lla lal lala la  lla lal lala la  lla lal lala la  lla lal lala la  lla lal lala la  lla lal lala la  lla lal lala la  lla lal lala la  lla lal lala la lalla la la lla la la lalallal a',
+    },
+  ].sort((a, b) =>
+    a.created_at > b.created_at ? 1 : b.created_at > a.created_at ? -1 : 0
+  );
```
Comment (x8er) [inline packages/frontapp/src/app/pages/HomePage.tsx:101]: You have to place this mock data in another place

**Review by IvanUglovets — APPROVED** [review]

**Review by sabrina34352 — APPROVED** [review]

**Pull request merged**